cmd/nerdctl/network_rm_linux_test.go

```diff
@@ -49,3 +49,20 @@ func TestNetworkRemove(t *testing.T) {
 	_, err = netlink.LinkByName("br-" + networkID[:12])
 	assert.Error(t, err, "Link not found")
 }
+
+func TestNetworkRemoveWhenLinkWithContainer(t *testing.T) {
+	t.Parallel()
+	if rootlessutil.IsRootless() {
+		t.Skip("test skipped for remove rootless network")
+	}
+	base := testutil.NewBase(t)
+	networkName := testutil.Identifier(t)
+
+	base.Cmd("network", "create", networkName).AssertOK()
```

Comment:
We need a test to remove a network after a container has exitted or stopped.

Reply:
make sense

Reply:
Done